beheer.md

```diff
@@ -104,13 +104,13 @@ docker service ps getstartedlab_web | Haal de gegevens op van alle tasks die de
 docker stack rm getstartedlab | Afsluiten van de applicatie.
 docker swarm leave --force | Afsluiten van de swarm.
 
-**Criteria**
-
-* API specificatie (OAS3) moet beschikbaar zijn en uit de imvertor genereerbaar zijn
-* Documentatie (zowel technisch als functioneel) moet beschikbaar zijn
-* De referentie-implementatie moet gereed en up-to date zijn met dan geldende OAS3
-* Het gebruikte Informatiemodel (minimaal UML)
-* Geautomatiseerde test en buildstraat (CI) moet beschikbaar voor de RI
-* De standaard moet ingebouwd zijn bij leveranciers (3 of meerdere leveranciers)
-* De standaard bevat een tabel waarin wordt aangegeven welke API versies met welke andere API versies compatible zijn.
+##Criteria##
+
+- API specificatie (OAS3) moet beschikbaar zijn en uit de imvertor genereerbaar zijn. Dit betekent dat zowel de Semantische Informatiemodellen (SIM), Uitwisselings Gegevensmodellen (UGM) als het Bericht Specificatiemodel (BSM) in UML beschikbaar moeten zijn.
+- Documentatie (zowel technisch als functioneel) moet beschikbaar zijn
+- De referentie-implementatie moet gereed en up-to date zijn met dan geldende OAS3
+- Het gebruikte Informatiemodel (minimaal UML)
```

Comment:
Het gebruikte Informatiemodel.... wat? Moet er zijn? Moet goedgekeurd zijn? Door wie? Moet overeenkomen met...?

Reply:
Het informatiemodel moet beschikbaar zijn en natuurlijk moet het goedgekeurd zijn. Ik ga er vanuit dat de governance rond API standaarden nog beschreven wordt en dat daarin wordt beschreven hoe en door wie API standaarden (en dus ook alle componenten die daar onderdeel van uit maken) goedgekeurd worden. Wie een informatiemodel goedkeurt is dus voor API Beheer niet zo van belang.